packages/databricks-vscode/resources/python/00-databricks-startup.py

```diff
@@ -0,0 +1,42 @@
+import os
+from pyspark.sql import SparkSession
+
+def loadEnv():
+    def findDatabricksDir(dirPath):
+        if(dirPath == "/"):
+            return None
+
+        subdirs = os.listdir(dirPath)
+        for dir in subdirs:
+            if(dir == ".databricks"):
+                return os.path.join(dirPath, ".databricks")
+
+        return findDatabricksDir(os.path.dirname(dirPath))
+
+    databricksDir = findDatabricksDir(os.path.abspath(os.curdir))
+
+    if "databricks.env" in os.listdir(databricksDir):
+        with open(os.path.join(databricksDir, "databricks.env"), "r") as fil:
+            for line in fil.readlines():
+                segs = line.split("=")
+                os.environ[segs[0]] = '='.join(segs[1:])
+
+loadEnv()
+del loadEnv
+
+spark = SparkSession.builder.getOrCreate()
```

Comment:
are these globals available in notebook cells?

Reply:
yes

Reply:
Nice